components/brave_wallet_ui/components/shared/create-network-icon/index.tsx

```diff
@@ -36,8 +49,14 @@ function CreateNetworkIcon (props: Props) {
   // Will remove these hardcoded chainId's.
   // We need to return a normal Ethereum Icon URL for Ethereum Mainnet
   // and return a grayed out Ethereum Icon Url for Ethereum Test Networks from the backend.
-  const testNetworkList = ['0x4', '0x3', '0x5', '0x2a', '0x539']
-  const needsPlaceholder = !['0x1', ...testNetworkList].includes(network.chainId) && (networkImageURL === '' || !isValidIcon)
+  const testNetworkList = [
+    BraveWallet.RINKEBY_CHAIN_ID,
+    BraveWallet.ROPSTEN_CHAIN_ID,
+    BraveWallet.GOERLI_CHAIN_ID,
+    BraveWallet.KOVAN_CHAIN_ID,
+    BraveWallet.LOCALHOST_CHAIN_ID
+  ]
+  const needsPlaceholder = nativeAsset.logo === '' && (networkImageURL === '' || !isValidIcon)
```

Comment:
Can we confirm this is still displaying a `Custom` networks `icon-url`?

Reply:
Nice catch! One of the cases below was not working. Here's how it looks after the fix.


| Network w/ preloaded icons | Network w/ custom icon URL + w/o asset icon URL | Network w/o icon URL + w/o asset icon URL | Network w/o icon URL + w/ custom asset icon URL |
-|-|-|-
|<img src="https://user-images.githubusercontent.com/3684187/155735964-4371fb2d-96a2-4bf6-837d-df05c5a35e3f.png">|<img src="https://user-images.githubusercontent.com/3684187/155736118-880ad51f-97dc-4570-9e80-01705ca0d6a2.png">|<img src="https://user-images.githubusercontent.com/3684187/155736630-9351d9b1-cd93-494d-84de-c3b9bb9d6237.png">|<img src="https://user-images.githubusercontent.com/3684187/155736606-8a7d56f2-9c1a-4a3b-9894-b1841d6f7fd8.png">|

Reply:
Looks good!